chroma-manager/chroma-manager.conf.template

```diff
@@ -1,6 +1,6 @@
 map $ssl_client_s_dn $ssl_client_s_dn_cn {
     default "";
-    ~/CN=(?<CN>[^/]+) $CN;
```

Comment:
what was the first forwardslash doing here?

Reply:
It was part of the old format which was forwardslash delimited and prefixed.  The new format is comma delimited with no leading comma.

Reply:
sp "...no leading forwardslash." I presume

Reply:
Not really.  To be clear, the old format was slashes with a leading slash and the new format is commas with no leading comma.